draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -204,14 +204,16 @@ const renderTag = (surveyStatus: SurveyStatus) => {
   if (surveyStatus.status === "default") {
     tagVariant = "default"
   }
-
-  return (
-    <div data-automation-id="survey-status-tag" className={styles.tag}>
-      <Tag variant={tagVariant} size="small">
-        {surveyStatus.text}
-      </Tag>
-    </div>
-  )
+  // TODO: explicitly omit/exclude profile as a variant type value
```

Comment:
2 things here:

I don't actually understand how `tagVariant` could ever be `"profile"`, it's a local variable to the `renderTag` function and never set to `profile`?
 
It'd be nicer to have a "guard statement/clause" here instead of wrapping the `return` statement in an `if`:
```
if (tagVariant == "profile" ) { return }

return (
  <div>...
```